Source/WebCore/platform/graphics/cv/ImageTransferSessionVT.mm

```diff
@@ -73,6 +73,26 @@
     m_pixelFormat = pixelFormat;
 }
 
+void ImageTransferSessionVT::setCroppingRectangle(std::optional<FloatRect> rectangle)
+{
+    if (m_croppingRectangle == rectangle)
+        return;
+
+    if (!rectangle) {
+        m_croppingRectangle = { };
+        m_sourceCroppingDictionary = { };
+        return;
+    }
+
+    m_croppingRectangle = rectangle;
```

Comment:
WTFMove(rectangle)

Can be move above considering this is always done (so no need for `m_croppingRectangle = { };`

Reply:
Will do